### PR TITLE
Ledger Eth Ropsten Debugging

### PR DESCRIPTION
**Pull request by AliMeer**
This PR does not represent changes that will be merged to dev and staging as it is, this change is for testing a particular defect on the dev environment hence the excessive logging.

### CODE REVIEW

File: src/lib/stark/ledger/createSignedTransfer.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ module.exports = async (
   if (transferTokenAddress) {
     const tokenInfo = byContractAddress(transferTokenAddress)
     transferTokenAddress = transferTokenAddress.substr(2)
+    console.log('tokenInfo ', tokenInfo)
```
Comment (plutoegg) [inline src/lib/stark/ledger/createSignedTransfer.js:34]: guess we dont need these console.logs

File: src/lib/stark/ledger/createSignedTransfer.js [hunk 2]
```diff
@@ -49,6 +51,9 @@ module.exports = async (
     transferTokenAddress = null
   }
 
+  console.log({transferTokenAddress, transferQuantization})
+  await eth.starkProvideQuantum(transferTokenAddress, transferQuantization)
```
Comment (plutoegg) [inline src/lib/stark/ledger/createSignedTransfer.js:55]: what level of testing has this had?

Comment (AliMeer) [reply src/lib/stark/ledger/createSignedTransfer.js:59]: @plutoegg I am keeping some console logging for this change as this is for debugging. console logging will be removed when we have tested.

File: src/lib/stark/ledger/createSignedTransfer.js (hunk 1 above)
Comment (AliMeer) [reply src/lib/stark/ledger/createSignedTransfer.js]: @plutoegg I am keeping some console logging for this change as this is for debugging. console logging will be removed when we have tested.

**Pull request merged**